src/components/atoms/button/styles.css

```diff
@@ -0,0 +1,20 @@
+.button {
+  border-radius: 30px;
+  padding: 15px 30px;
+  color: white;
+  background-color: #105EFB;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  font-size: 0.6rem;
```

Comment:
You used `px` in all others properties, maybe it's good to use here too in order to avoid side effects.